locales/en/messages.json

```diff
@@ -6220,6 +6220,14 @@
         "message": "<span class=\"message-negative\">Error</span> while saving the VTX Config file",
         "description": "Message in the GUI log when the VTX Config file is saved"
     },
+    "vtxSavedLuaFileOk": {
+        "message": "lua VTX Config file <span class=\"message-positive\">saved",
```

Comment:
You need to put the opening span before the text you want to change the color, and the closing span after it.